src/coreclr/jit/emit.h

```diff
@@ -768,12 +768,26 @@ class emitter
         unsigned _idLargeDsp : 1;  // does a large displacement follow?
         unsigned _idLargeCall : 1; // large call descriptor used
 
-        unsigned _idBound : 1; // jump target / frame offset bound
-#ifndef TARGET_ARMARCH
-        unsigned _idCallRegPtr : 1; // IL indirect calls: addr in reg
-#endif
-        unsigned _idTlsGD : 1; // Used to store information related to TLS GD access on linux
-        unsigned _idNoGC : 1;  // Some helpers don't get recorded in GC tables
+        // We have several pieces of information we need to encode but which are only applicable
+        // to a subset of instrDescs. To accommodate that, we define a several _idCustom# bitfields
+        // and then some defineds to make accessing them simpler
```

Comment:
These bits are "expensive" and impact the maximum size of "small" constants, so I opted to repurpose these existing 3 bits that are only used for `IF_LABEL`, `IF_METHOD`, and related formats. They will never conflict with the SIMD instructions so this ends up being a nice way to fit it in, IMO.